src/components/account-header/account-header.yaml

```diff
@@ -87,6 +87,7 @@ examples:
       classes: 'the-header-classes'
   visualRegressionTests:
     backstopScenarioOptions:
+      delay: 2000
```

Comment:
@TimothyFothergill sorry because this wasn't well described / linked on the ticket, the wait that I was originally thinking of adding was copying this waitFor to all the scenarios that open the menu

https://github.com/hmrc/hmrc-frontend/blob/252adf3e85aaa06ee8d103132bbc733bc169650c/src/components/account-header/account-header.yaml#L165

it might be worth checking when the delay added here will occur, if it happens after interactions like click/hover then it should fix the flakiness too, but if it happens before then it might not